scripts/binary/binary-integrity-check-source.js

```diff
@@ -2,6 +2,8 @@ const OrigError = Error
 const captureStackTrace = Error.captureStackTrace
 const toString = Function.prototype.toString
 const callFn = Function.call
+const filter = Array.prototype.filter
+const startsWith = String.prototype.startsWith
```

Comment:
Do you know why we create all these aliases in the first place instead of just doing `tempError.filter(...`) etc? It looks like something to do with the integrity check - how does comparing `if (filter.call !== callFn) {` verify the integrity?

Reply:
We are protecting against the prototype call method on each of those functions being tampered with. So we want to make sure that they are still equal to the original function call prior to using them